app/views/insured/families/_enrollment_refactored.html.erb

```diff
@@ -115,13 +115,13 @@
             <%= render partial: "shared/plan_shoppings/sbc_link", locals: { plan: product, custom_css: false, hbx_id: hbx_enrollment.hbx_id } %>
             <% unless details_page %>
               <a
-                class="ml-4"
+                class="ml-4 plan-contact-info-alt"
                 id="plan_contact_info-<%= hbx_enrollment.hbx_id%>"
                 data-toggle="modal"
                 data-target="#<%= product.kind.to_s %>-<%= product.id %>-<%= hbx_enrollment.hbx_id%>"
                 tabindex="0"
                 href="#"
-                onkeydown="handleContactInfoKeyDownAlt(event, 'plan_contact_info-<%= hbx_enrollment.hbx_id%>', '#<%= product.kind.to_s %>-<%= product.id %>-<%= hbx_enrollment.hbx_id%>')"
+                data-click-id="plan_contact_info-<%= hbx_enrollment.hbx_id%>"
```

Comment:
Should this be `data-keydown-id`?

Reply:
changed click-id element name to be more descriptive